examples/federated_analysis/configs/fed_analysis/custom/analysis_executor.py

```diff
@@ -40,6 +37,7 @@ def __init__(
         n_bins: int = 256,
         range_min: float = 0.0,
         range_max: float = 255.0,  # assumes BMP/PNG/JPEGSs as input
+        supported_task: str = SupportedTasks.HISTOGRAM,
```

Comment:
See my previous comment about the arg name 'supported_task'.

Reply:
So, in my understanding, the controller sends a task out, hence just "task_name_histogram" but the executor only can support certain tasks. Hence, "supported_task" in its argument. I wanted to make that distinction by those argument names.